std_msgs/Header header 				# for time stamp
geometry_msgs/Pose2D robotPose        		# Pose of the robot according to the particle filter localization
float32 robotPoseConfidence
